pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -758,3 +740,79 @@ func mergeEnvVars(original, new []corev1.EnvVar) []corev1.EnvVar {
 
 	return original
 }
+
+func setProbes(customLiveness corev1.Probe, customReadiness corev1.Probe) (*corev1.Probe, *corev1.Probe) {
+	liveness := &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path: "/api/v4/system/ping",
+				Port: intstr.FromInt(8065),
+			},
+		},
+		InitialDelaySeconds: 10,
+		PeriodSeconds:       10,
+		FailureThreshold:    3,
+	}
+
+	if customLiveness.Handler != (corev1.Handler{}) {
+		liveness.Handler = customLiveness.Handler
```

Comment:
Do we want users to be able to set their own handlers? I feel like being able to set things like the path and port could lead to unexpected pod behavior. I am all for being able to set things like failureThreshold, but I am not sure I see the value in being able to set the handler at this time.

Thoughts?

Reply:
no strong opinions, but imagine the user create a custom image and there they define a custom health endpoint in another path or even using a script or any other method. then will be not possible. was in this specific case i was thinking to have the entire probe object and also to help a bit in the development :)

but if you dont want that and is better to have just the other fields I'm happy to change. 

@gabrieljackson

Reply:
That is something I hadn't considered, but makes a lot of sense! Thanks.

Reply:
my bad, i should improve the PR description.
mental note to myself